### DISCUSSION TITLE
more than >35% of all bug reports are still open, this project is over a year old, time to suck a lot less.

### DISCUSSION THREAD

**Discussion by sovos15** [civitai/civitai#1413] [Ideas]
Rather than paying people to gawk at problems they don't know how to fix. Offer money to people who can fix it. Rather than sallaries and hourly pay, peace mail can save you from this hell. 

The first step to solving a problem is knowing you have one. Ideally before someone starts hitting you with it.
1. [HackerOne ](https://www.hackerone.com/)
2. [Bugcrowd](https://www.bugcrowd.com/)
3. [Intigriti](https://www.intigriti.com/)
4. [Open Bug Bounty](https://www.openbugbounty.org/)  
5. [HackenProof](https://hackenproof.com/blog/product-news/what-is-hackenproof)

I know the idea of paying for hard work is toxic and paying for nothing is in vouge but stop that. It's stupid. 

Now that you know what the problems are before I do, get them fixed before I can bitch about it.
1. [Droomwork](https://www.droomwork.io/)
2. [Upwork](https://www.upwork.com/)
3. [Toptal](https://www.toptal.com/)
4. [Peopleperhour](https://www.peopleperhour.com/)
5. [Fiverr](https://www.fiverr.com/)

Just like that I'm no longer here and angry at you.

**Comment by omarsoufiane** [comment]
I apologize if this comment comes across as self-promotion, but I’ve developed a platform called [BountyHub](https://bountyhub.dev), which is designed specifically for situations like this.

**Reply by sovos15** [reply]
Sad to me that self promotion has become a dirty word with the level of information control those that would like to charge for exposure have developed. The glass celling just keeps encroaching.
